packages/front-end/components/InitialSetup/InitiateConnectionPage.tsx

```diff
@@ -0,0 +1,45 @@
+import { RadioGroup } from "@radix-ui/themes";
+import { UseFormReturn } from "react-hook-form";
+import SDKLanguageSelector from "@/components/Features/SDKConnections/SDKLanguageSelector";
+import { SdkFormValues } from "@/pages/setup";
+
+interface Props {
+  connection: string | null;
+  form: UseFormReturn<SdkFormValues, unknown>;
+}
+
+const InitiateConnectionPage = ({ connection, form }: Props) => {
```

Comment:
When you jump back into the setup flow should this pre-fill with the first SDK connection created or should we store the SDK connection ID somewhere when someone passes this step?

Reply:
Went with the former